crates/router/src/connector/paypal/transformers.rs

```diff
@@ -21,6 +23,9 @@ use crate::{
     },
 };
 
+#[cfg(feature = "payouts")]
+const LANGUAGE: &str = "en-US";
```

Comment:
Possible to move this to consts crate? And name it better? `PAYPAL_NOTIFICATION_SCRIPT_LANGUAGE` ?